### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by vrothberg**
Add a `Communications` section to the README to make it more
approachable for new users who want to reach the community and
maintainers.

@TomSweeneyRedHat PTAL

### CODE REVIEW

**Comment by rhatdan** [comment]
LGTM

**Comment by mheon** [comment]
LGTM

**Comment by giuseppe** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -27,6 +27,20 @@ At a high level, the scope of libpod and Podman is the following:
...
+
+For discussions around issues/bugs and features, you can use the GitHub
+[issues](https://github.com/containers/libpod/issues)
+and
+[PRs](https://github.com/containers/libpod/pulls)
+tracking system.
+
+There is also a [mailing list](https://lists.podman.io/archives/) at `lists.podman.io`.
+You can subscribe by sending a message to `podman@lists.podman.io` with the subject `subscribe`.
+
```
Comment (TomSweeneyRedHat) [inline README.md]: I'd move the roadmap section up above the new communications section.  It should come immediately after the Overview and Scope section.

**Comment by TomSweeneyRedHat** [comment]
LGTM other than the location of the section in the doc.

**Comment by mheon** [comment]
@vrothberg Rebase please

**Comment by vrothberg** [comment]
Done :heavy_check_mark:

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
Happy green test buttons if you want to cancel the hold @mheon.

**Pull request merged**